.github/workflows/global-ci.yaml

```
@@ -59,13 +56,40 @@ jobs:
         podman build --build-arg SEED_ROOT=/opt/app-root/src/tackle2-seed -t $IMG .
         podman push $IMG
 
+  build-kantra:
+    name: Build kantra
+    runs-on: ubuntu-24.04
+    strategy:
+      fail-fast: false
+    steps:
+    - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)